src/main/java/mate/academy/model/CartItem.java

```diff
@@ -0,0 +1,37 @@
+package mate.academy.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity(name = "cart_item")
```

Comment:
table name should be plural

```suggestion
@Entity(name = "cart_items")
```